test/e2e/redundancy/gateway_failover.go

```diff
@@ -32,8 +32,54 @@ var _ = Describe("[redundancy] Gateway fail-over tests", func() {
 			testGatewayFailOverScenario(f)
 		})
 	})
+
+	When("a Route-agent pod is restarted on the Gateway node", func() {
+		It("should start a new submariner engine pod, and be able to connect from another cluster", func() {
+			testRouteAgentFailOverScenario(f)
+		})
+	})
+
```

Comment:
This should not be grouped with the gateway fail-over tests.  Please add a new top-level `Describe("[redundancy] Route Agent fail-over tests"...)` in a separate _route_agent_failover.go_ file.